release/internal/hashrelease/hashrelease.go

```diff
@@ -97,25 +97,63 @@ func ListHashreleases(sshConfig *command.SSHConfig) ([]hashrelease, error) {
 	return folders, nil
 }
 
+func getHashreleaseLibrary(sshConfig *command.SSHConfig) (string, error) {
+	out, err := command.RunSSHCommand(sshConfig, fmt.Sprintf("cat %s", remoteReleasesLibraryPath()))
+	if err != nil {
+		logrus.WithError(err).Error("Failed to get hashrelease library")
+		return "", err
+	}
+	return out, nil
+}
+
+func cleanHashreleaseLibrary(sshConfig *command.SSHConfig, hashreleaseNames []string) error {
+	library, err := getHashreleaseLibrary(sshConfig)
+	if err != nil {
+		return err
+	}
+	scanner := bufio.NewScanner(strings.NewReader(library))
+	var newLibrary []string
+	for scanner.Scan() {
+		line := scanner.Text()
+		for _, name := range hashreleaseNames {
+			if !strings.Contains(line, name) {
```

Comment:
I considered that but it will still result in making a diff because the content is in the format "<hash> - <note>" which we do have access to recreate